README.md

```diff
@@ -54,7 +54,7 @@ user interfaces that can appear in any language and support the needs of diverse
 
 The current specification starts [here](spec/README.md) and may have changed since the publication
 of the Tech Preview version.
-The Tech Preview specification is [here](tr35-messageformat.md) (link to follow).
+The Tech Preview specification is [here](https://www.unicode.org/reports/tr35/tr35-messageFormat.html)
```

Comment:
This link will go to whatever the current released version of LDML is. Great if that's what's intended.

If you want the exact version released with CLDR 45 it is here: <https://www.unicode.org/reports/tr35/tr35-72/tr35-messageFormat.html>